### PR TITLE
feat: check field/sortField/sortOrder in ReadRequestFromQueryParams

### PR DESCRIPTION
**Pull request by dplynskiy**
ASP-976

### CODE REVIEW

File: controllers/base.go [hunk 1]
```diff
@@ -71,6 +71,18 @@ func (c *ApiController) ReadRequestFromQueryParams() BaseDataManageRequest {
 		Organization: c.Input().Get("organization"),
 	}
 
+	if !util.FilterField(result.Field) {
+		result.Field = ""
```
Comment (ISulimanov) [inline controllers/base.go:75]: should we clean up value also in this case?

File: controllers/base.go [hunk 2]
```diff
@@ -71,6 +71,18 @@ func (c *ApiController) ReadRequestFromQueryParams() BaseDataManageRequest {
 		Organization: c.Input().Get("organization"),
 	}
 
+	if !util.FilterField(result.Field) {
```
Comment (svgrafov) [inline controllers/base.go:74]: Name of the FilterField() seems off.
Looks like it must be IsFieldValueAllowedForDB or something like that.

**Review by svgrafov — CHANGES_REQUESTED** [review]
Please add tests for both ReadRequestFromQueryParams() and FilterField(), since this a security fix.

File: controllers/base.go (hunk 1 above)
Comment (dplynskiy) [reply controllers/base.go:75]: yes, done.

File: controllers/base.go (hunk 2 above)
Comment (dplynskiy) [reply controllers/base.go]: done

**Comment by dplynskiy** [comment]
> Please add tests for both ReadRequestFromQueryParams() and FilterField(), since this a security fix.

added tests for IsFieldValueAllowedForDB but it's hard for ReadRequestFromQueryParams until we haven't DI here

**Review by ISulimanov — APPROVED** [review]

**Review by svgrafov — APPROVED** [review]

**Review by ISulimanov — APPROVED** [review]

**Review by svgrafov — APPROVED** [review]

**Pull request merged**